src/transactions.rs

```diff
@@ -1151,5 +1222,11 @@ mod tests {
         cancel_tx.hex().expect("Hex repr cancel_tx");
         emergency_tx.hex().expect("Hex repr emergency_tx");
         feebump_tx.hex().expect("Hex repr feebump_tx");
+
+        // Test serialize method for a transaction
+        let serialized_tx = serde_json::to_string(&vault_tx).unwrap();
+        let deserialized_tx = serde_json::from_str(&serialized_tx).unwrap();
+
+        assert_eq!(vault_tx, deserialized_tx);
```

Comment:
I was thinking of a new test function doing roundtrips (ser / deser) for all transactions

Reply:
yes, will do that